package.json

```diff
@@ -20,9 +20,8 @@
     "test": "yarn validate:examples && yarn lint && yarn flow check --max-warnings 0 && yarn tsc && yarn unit-test",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook -o public/legacy && build-storybook",
-    "storybook:screener:wrapper": "./scripts/run-screener.sh",
-    "storybook:screener": "yarn run storybook:screener:wrapper --conf screener.config.js",
-    "storybook:screener:dark": "THEME=dark yarn run storybook:screener:wrapper --conf screener.config.js",
+    "storybook:screener": "yarn run screener-storybook --conf screener.config.js --debug",
```

Comment:
Do we want to keep the `--debug` flag here?

Reply:
ops, no - thanks Nadiia